src/Hit/Git.hs

```diff
@@ -40,8 +41,18 @@ runNew issueNum = do
     if login == ""
         then errorMessage "user.login is not specified"
         else do
-            let branchName = login <> "/" <> show issueNum
+            let issueId = mkIssueId issueNum
+            issueTitle <- getIssueTitle issueId
```

Comment:
I like that now it won't be possible to create the branch with the not-valid issue id!